components/molecule/buttonGroup/src/index.scss

```diff
@@ -0,0 +1,7 @@
+@import "~@schibstedspain/sui-atom-button/lib/index";
```

Comment:
There shouldn't be any atom button dependency. The styles should be loaded in the component which uses the button group using atom buttons as child. It would be top if the button group component would work flawless even for html buttons as childs